app/assets/stylesheets/components/_forms.scss

```diff
@@ -1,21 +1,25 @@
 .form-inputs {
   width: 300px;
+  margin-bottom: 15px;
 }
 .various-forms{
   display: flex;
   flex-direction: column;
   align-items: center;
   padding-top: 30px;
 }
-input.btn.button-form {
+.button-form {
```

Comment:
Awesome/ Did you have the chance to check that this styles also work on the forms generated by devise?